llvm/include/llvm/CodeGen/GlobalISel/GenericMachineInstrs.h

```diff
@@ -868,6 +868,14 @@ class GZext : public GCastOp {
   };
 };
 
+/// Represents an any ext.
+class GAnyExt : public GCastOp {
```

Comment:
You also need to add a case to the switch in `GCastOp`

Reply:
`CastOp` already has `G_ANYZEXT` or why misunderstood your comment.

Reply:
Indeed, I missed that.